src/Util/SearchResult.php

```diff
@@ -135,7 +135,21 @@ public function fetchNext()
     {
         $searchResult = null;
 
-        if (array_key_exists('scrollId', $this->options)) {
+        if (array_key_exists('size', $this->options)
+            && array_key_exists('sort', $this->filters)
+            && !array_key_exists('from', $this->options)) {
```

Comment:
Why not use search after if `from` is set? Removing it from the search after request should still be consistent.